src/main/java/entities/HumanPlayer.java

```diff
@@ -42,4 +42,19 @@ public void incrementLosses() {
         this.losses ++;
     }
 
+    /**
+     * Get the number of wins of this HumanPlayer.
+     * @return An int containing the number of wins of this HumanPlayer.
+     */
+    public int getWins() {
```

Comment:
this is technically unnecessary because of the getStats() method here, however i prefer distinct methods as opposed to putting everything into an array. we should decide which behaviour to stick with

Reply:
This behaviour is better I believe.

Reply:
Okay I'll be removing getStats(). There are a few tests in HumanPlayerTest that use the getStats() method so I'll be fixing it in my following commits.